release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -1729,6 +1741,40 @@ In the following tables, features are marked with the following statuses:
 |Not Available
 |Technology Preview
 
+|MT2892 Family [ConnectX-6 Dx] SR-IOV support
+|Not Available
+|Not Available
+|Technology Preview
+
+|MT2894 Family [ConnectX-6 Lx] SR-IOV support
+|Not Available
+|Not Available
+|Technology Preview
+
+|MT42822 BlueField-2 in ConnectX-6 NIC mode SR-IOV support
+|Not Available
+|Not Available
+|Technology Preview
+
+|Silicom STS Family SR-IOV support
+|Not Available
+|Not Available
+|Technology Preview
+
+|MT2892 Family [ConnectX-6 Dx] OvS Hardware Offload support
```

Comment:
I would remove this entry since:
https://github.com/openshift/openshift-docs/pull/54432
correctly only mentions CX-6 Lx and BF2 in NIC mode.